### ISSUE TITLE
MF: On launching the app a blank page gets displayed at the start

### ISSUE THREAD

**Issue by xmano** [blockframes/blockframes#5330]
![image](https://user-images.githubusercontent.com/8731418/111975432-9af99c80-8b26-11eb-894f-da46ca1df99a.png)

I see that the Google Map script is time outing at my end. So it could be cause of blank screen.

We could see if async/defer of the script can improve the first paint here.

**Comment by fritzschoff** [comment]
can't reproduce this error. Also there is no error on the current staging. Only error you got is when you are using brave and it blocks the googletagmanager requests, but this is client side. @xmano can we close it?

**Comment by xmano** [comment]
May have been fixed.. @Mussuranae was to check on this. Will ask her to close if she did any change.

**Comment by Mussuranae** [comment]
I haven't this error also. Tried to reproduce it on local and on staging but can't. I checked if the google map API key allows Media Financiers to use it. And the answer is yes. The key allows the localhost as well, so in our env, it has to work. Also, I don't understand why the google map is trigger on this page as the only map we have is on the contact page (link in the footer).

![image](https://user-images.githubusercontent.com/48033264/113278254-00494c80-92e2-11eb-9030-ad98da6ab961.png)

Is the error really related to the google map ? I think we can close this issue if there is no clue to solve or it is not on prod/staging.
